types/functions.d.ts

```diff
@@ -0,0 +1,14 @@
+type navigate = (
+  to: string,
+  {
+    replace,
+    state
+  }: {
```

Comment:
This options object as well as its `state` property should be optional, shouldn't they?

Reply:
Thanks for pointing this one, went unnoticed by me